cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -204,6 +204,32 @@ def test_get_dictionary_mapping_curie(self):
             self.writer._get_mapping_dict_curie(ids, curie_constraints)
 
 
+class TestIgnoreLabelFunctions(unittest.TestCase):
+    
+    def setUp(self):
+        #Set up test data
+        self.test_adata = adata
+        self.test_adata_with_labels = adata_with_labels
+
+    def test_validating_labeled_h5ad_should_fail_if_no_flag_set(self):
+
+        validator = Validator()
+        validator.adata = self.test_adata_with_labels
+        is_valid = validator.validate_adata()
+
+        self.assertFalse(is_valid)
+
+    def test_validating_labeled_h5ad_should_pass_if_flag_set(self):
+
+        validator = Validator(ignore_labels=True)
+        import copy
+        validator.adata = copy.deepcopy(self.test_adata_with_labels)
+        validator.adata.uns["X_normalization"] = 'none'
```

Comment:
Won't pass validation if `X_normalization` isn't `none`.

Reply:
Also remember that we're deprecating `X_normalization` in 3.0.0.

Reply:
Good point. I feel even less guilty about this hack now.